src/test/java/oryanmoshe/kafka/connect/util/TimestampConverterTests.java

```diff
@@ -101,9 +116,7 @@ void converterTest(final String columnType, final String format, final String in
         Object actualResult = mockRegistration._converter.convert(input);
         System.out.println(actualResult);
         if (actualResult == null) {
```

Comment:
```suggestion
        if (actualResult == null) {
```
Honestly don't know if its the Java way
But if we do 
if (actualResult != null)
we can skip the if else 
WDYT?

Reply:
u right, fixed